### PR TITLE
testing/py3-httpbin: new aport

### PR DESCRIPTION
**Pull request by z3ntu**

### CODE REVIEW

File: testing/py3-brotlipy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Maintainer: Luca Weiss <luca@z3ntu.xyz>
+pkgname=py3-brotlipy
+_pyname=brotlipy
+pkgver=0.7.0
+pkgrel=0
+pkgdesc="Python binding to the Brotli library"
+url=https://github.com/python-hyper/brotlipy
+arch=all
```
Comment (maxice8) [inline testing/py3-brotlipy/APKBUILD]: quoted

File: testing/py3-brotlipy/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+license=MIT
```
Comment (maxice8) [inline testing/py3-brotlipy/APKBUILD]: quoted

File: testing/py3-brotlipy/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Maintainer: Luca Weiss <luca@z3ntu.xyz>
+pkgname=py3-brotlipy
+_pyname=brotlipy
+pkgver=0.7.0
+pkgrel=0
+pkgdesc="Python binding to the Brotli library"
```
Comment (maxice8) [inline testing/py3-brotlipy/APKBUILD]: suggestion: Python3

File: testing/py3-brotlipy/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+url=https://github.com/python-hyper/brotlipy
+arch=all
+license=MIT
+depends="py3-cffi"
+makedepends="python3-dev"
+_pypiprefix="${_pyname%${_pyname#?}}"
+source="https://files.pythonhosted.org/packages/source/$_pypiprefix/$_pyname/$_pyname-$pkgver.tar.gz"
+builddir=$srcdir/$_pyname-$pkgver
```
Comment (maxice8) [inline testing/py3-brotlipy/APKBUILD]: quoted

File: testing/py3-brotlipy/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+_pyname=brotlipy
+pkgver=0.7.0
+pkgrel=0
+pkgdesc="Python binding to the Brotli library"
+url=https://github.com/python-hyper/brotlipy
+arch=all
+license=MIT
+depends="py3-cffi"
+makedepends="python3-dev"
+_pypiprefix="${_pyname%${_pyname#?}}"
```
Comment (maxice8) [inline testing/py3-brotlipy/APKBUILD]: imo it is cleaner to do ${pkgname:0:1}

File: testing/py3-brotlipy/APKBUILD (hunk 1 above)
Comment (z3ntu) [reply testing/py3-brotlipy/APKBUILD]: Then fix https://wiki.alpinelinux.org/wiki/Python_package_policies :stuck_out_tongue:

Comment (maxice8) [reply testing/py3-brotlipy/APKBUILD]: fixed

**Comment by z3ntu** [comment]
I've removed py3-brotlipy now because that project is basically dead upstream and httpbin can also work with the upstream brotli python bindings (which are now built).

Any idea why the CI fails though?

**Comment by maxice8** [comment]
our setup is notoriously bad in dealing with changes in the same Pull request, your changes to brotli aren't propagating to when py3-httpbin is built.

i'll merge the brotli changes separately then you can re-push this branch and hopefully they will be there

**Comment by maxice8** [comment]
Ok i pushed the brotli changes, please rebase and after 5 or 10 minutes push again so the CI tries again

**Pull request closed without merging**